### PR TITLE
[zh]Update content/zh/docs/reference/command-line-tools-reference/kub…

### PR DESCRIPTION
**Pull request by zyy19981018**

### CODE REVIEW

File: content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md [hunk 1]
```diff
@@ -1287,7 +1244,7 @@ TopologyManager=true|false (BETA - default=true)<br/>
 VolumeCapacityPriority=true|false (ALPHA - default=false)<br/>
 WinDSR=true|false (ALPHA - default=false)<br/>
 WinOverlay=true|false (BETA - default=true)<br/>
-WindowsHostProcessContainers=true|false (ALPHA - default=false)
+WindowsHostProcessContainers=true|false (BETA - default=true)
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md:1247]: 此列表变化较大，请仔细核对。

File: content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md [hunk 2]
```diff
@@ -2279,38 +2173,17 @@ This can be used to allow load balancer to stop sending traffic to this server.
...
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-logs at or above this threshold go to stderr
+If true the HTTP Server will continue listening until all non long running request(s) in flight have been drained, during this window all incoming requests will be rejected with a status code 429 and a 'Retry-After' response header, in addition 'Connection: close' response header is set in order to tear down the TCP connection when idle.
 -->
-将达到或超过此阈值的日志写到标准错误输出
+如果为 true，则 HTTP 服务器将继续侦听，直到所有非长时间运行的请求都被耗尽，
+在此窗口期间，所有传入请求将被拒绝，状态代码为 429 和 "Retry-After" 响应标头，
+此外， "Connection: close"响应头设置是为了在空闲时断开 TCP 连接。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md:2186]:
```suggestion
此外，"Connection: close" 响应头设置是为了在空闲时断开 TCP 连接。
```

**Comment by tengqm** [comment]
/hold
@zyy19981018 have you carefully reviewed the list for feature gates?

**Comment by tengqm** [comment]
This needs to be rebased on to main. There is a conflict to be resolved.

**Comment by Sea-n** [comment]
Hey @zyy19981018, are you ready to move forward?
Please rebase onto `kubernetes:main`. If you have any problem, you can contact us though `#kubernetes-docs-zh` channel in [Slack](https://slack.k8s.io/).

**Comment by tengqm** [comment]
/close
Stale for too long and there are conflicts to be resolved.
Please feel free to reopen and resume this work.

**Pull request closed without merging**